core/src/main/java/discord4j/core/object/audit/ChangeKey.java

```diff
@@ -16,14 +16,14 @@
  */
 package discord4j.core.object.audit;
 
+import discord4j.common.util.Snowflake;
 import discord4j.core.object.ExtendedPermissionOverwrite;
 import discord4j.core.object.entity.Guild;
 import discord4j.core.object.entity.Role;
 import discord4j.core.object.entity.channel.Channel;
 import discord4j.rest.util.PermissionSet;
-import discord4j.common.util.Snowflake;
 
-import java.awt.*;
+import java.awt.Color;
```

Comment:
Hmm.. this is unrelated to this PR but just noticed we still depend on java.awt.Color here

Reply:
Fixed in https://github.com/Discord4J/Discord4J/pull/862